test/test-analytics.js

```diff
@@ -12,6 +12,12 @@ var appc = require('../index'),
 	temp = require('temp');
 
 describe('analytics', function () {
+
+	beforeEach(done => {
+		appc.analytics.events.splice(0, appc.analytics.events.length);
```

Comment:
The previous assignment to the exported array didn't work, we need to actually operate on the export array via a method like splice here